packages/module/src/catalog/QuickStartTile.tsx

```diff
@@ -90,6 +90,7 @@ const QuickStartTile: React.FC<QuickStartTileProps> = ({
           />
         }
         onClick={handleClick}
+        href="#"
```

Comment:
added due to CatalogTile bug
https://github.com/patternfly/patternfly-react/issues/7039